docs/components/index.rst

```diff
@@ -0,0 +1,17 @@
+base_components
```

Comment:
Please Name it Components instead of base_components

Reply:
Change made here: 90e4dd4